src/CFamily.UnitTests/Subprocess/ProcessRunnerTests.cs

```diff
@@ -26,6 +26,7 @@
 using System.Threading.Tasks;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarLint.VisualStudio.CFamily.Subprocess;
```

Comment:
I haven't changed the namespaces of the test and product files yet, so they don't match the namespace of the generated resources file -> this `using` is required so the resources strings are accessible.